config/locales/en.yml

```diff
@@ -85,9 +85,31 @@ en:
       title: Frequently Asked Questions
   helpers:
     channels:
-      generic_verification_failure: Channel could not be verified!
       verification_in_progress: Verification in progress
-      verification_awaiting_admin_approval: Awaiting admin approval
+      verification_awaiting_admin_approval: Under review
+      verification_failure_what_happened: What happened
+      verification_failure: Your channel could not be verified!
+      verification_failure_explanation:
+        domain_not_found: "we could not find your domain."
+        connection_failed: |
+          we could not find the public file at https://%{domain}/.well-known/brave-payments-verification.txt.
+        too_many_redirects: "we were redirected too many times to verify your domain. "
+        no_txt_records: "there are no TXT records on your domain's DNS. "
+        token_incorrect_dns: "the token in your TXT records is incorrect. "
+        token_not_found_dns: "we could not find the token in any of the TXT records. "
+        token_not_found_public_file: "we could not find the token in your public file. "
+        no_https: "HTTPS is not enabled. "
+        generic: "of an unkown reason. "
+      verification_failure_cta:
+        domain_not_found: "Please check if the domain is correct."
+        connection_failed: "Please make sure the file exists and is publicly available."
+        too_many_redirects: "Please reduce the number of redirects."
+        no_txt_records: "Please make sure you have entered the token in your DNS records:"
+        token_incorrect_dns: "Please make sure the token in your DNS entry is correct."
+        token_not_found_dns: "Please add this TXT entry to your DNS records:"
+        token_not_found_public_file_html: "Please make sure the verification token is present in the public file %{domain}/.well-known/brave-payments-verification.txt."
+        no_https: "Please enabled HTTPS on your domain, or choose another verification method."
+        generic: "Please contact %{support_email}."
```

Comment:
![image](https://user-images.githubusercontent.com/8647607/43503636-eb04d740-9514-11e8-9b7f-22190f6516b0.png)

To be honest, the CTA is confusing since there's 2 conflicting CTAs (1. please email us and 2. Retry Verification). I think we should simplify this somehow, or even adjust the wording to say "If this verification failure continues to occur, please reach us at <email>".